platform/ios/PolyPodApp/Features/FeatureStorage.swift

```diff
@@ -86,6 +86,8 @@ final class FeatureStorage {
 
         var categories: [FeaturesCategoryModel] = []
         for metaCategory in metaCategories {
+            guard !metaCategory.features.isEmpty else { continue }
```

Comment:
This is not totally related to the PR, but I guess it's a guard clause in case there are no features there; it could happen perfectly. However, in that case, shouldn't we guard also against `features` not being there? Some might see there's some empty {} and just eliminate the corresponding key. I'm not totally sure we statically check for that somewhere; it that's the case, this probably wouldn't be needed.